src/libclang/libclang_parser.cpp

```diff
@@ -373,6 +375,12 @@ void libclang_compile_config::do_set_flags(cpp_standard standard, compile_flags
         else
             add_flag("-std=c++1z");
         break;
+    case cpp_standard::cpp_20:
```

Comment:
Please compare the clang version with the selected flag and raise an error if it doesn't support C++20 yet. I think clang 6 is the first one that supports `-std=c++20`?